tests_new/integration_tests/modules/share_base/fresh_deployment.py

```diff
@@ -0,0 +1,106 @@
+from assertpy import assert_that
+
+from dataall.modules.shares_base.services.shares_enums import ShareItemStatus, ShareObjectStatus
+from tests_new.integration_tests.errors import GqlError
+
+from tests_new.integration_tests.modules.share_base.queries import (
+    submit_share_object,
+    add_share_item,
+    get_share_object,
+    reject_share_object,
+    approve_share_object,
+    revoke_share_items,
+)
+from tests_new.integration_tests.modules.share_base.utils import check_share_ready
+
+
+def test_create_share_object(share1):
+    assert_that(share1.status).is_equal_to(ShareObjectStatus.Draft.value)
+
+
+def test_submit_empty_object(client2, share1):
+    # here Exception is not recognized as GqlError, so we use base class
+    # toDo: back to GqlError
+    assert_that(submit_share_object).raises(Exception).when_called_with(client2, share1.shareUri).contains(
+        'ShareItemsFound', 'The request is empty'
+    )
+
+
```

Comment:
Once the dataset PRs are merged we should add tests on getShareObject with the filters used to list shared items. Making sure it lists the tables, folders and bucket defining them explicitly instead of depending on a list of items